Source/Classes/AnimatedImages/PINAnimatedImage.m

```diff
@@ -78,24 +96,25 @@ - (NSTimeInterval)minimumFrameInterval
     return (scaledGCD / kGreatestCommonDivisorPrecision);
 }
 
-//Credit to FLAnimatedImage ( https://github.com/Flipboard/FLAnimatedImage ) for display link interval calculations
+// In practice, a recursive method should be fine because we're never looking for a really
+// large gcd.
 static NSUInteger gcd(NSUInteger a, NSUInteger b)
```

Comment:
It's correct we only use it in testing right? Otherwise maybe we should move to a non recursive version?

Reply:
Switched it.